src/applications/mhv-landing-page/components/alerts/AlertMhvNoAction.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+// eslint-disable-next-line import/no-named-default
+import { default as recordEventFn } from '~/platform/monitoring/record-event';
+import { VaAlert } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+
+const AlertMhvNoAction = ({ errorCode, testId, recordEvent }) => {
+  const headline = `You can't access messages, medications, or medical records right now`;
```

Comment:
We have the `title` prop for this component. Do we want to use that for headline?
 `title: "You can't access messages, medications, or medical records right now",`